MM2SpaceCenter.roboFontExt/lib/MM2SpaceCenter.py

```diff
@@ -15,139 +15,147 @@
 from defconAppKit.windows.baseWindow import BaseWindowController
 from mojo.events import addObserver, removeObserver, postEvent
 from vanilla import *
+from lib.UI.integerEditText import NumberEditText
 from mojo.extensions import *
 import codecs
 
 class MM2SpaceCenter:
     
     def activateModule(self):
         addObserver(self, "MMPairChangedObserver", "MetricsMachine.currentPairChanged")
+        
         print ('MM2SpaceCenter activated')
 
+
     def deactivateModule(self, sender):
         removeObserver(self, "MetricsMachine.currentPairChanged")
+        removeObserver(self, "FutureProof.currentPairChanged")
```

Comment:
I don't know what this is. Github was being unintuitive, so I enacted my UI updates upon the latest version of the file I could find.